src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseData.java

```diff
@@ -249,7 +249,8 @@ public class CaseData extends CaseDataParent {
 
     private final String boEmailGrantIssuedNotificationRequested;
 
-    private final String boEmailGrantIssuedNotification;
+    @Builder.Default
+    private final String boEmailGrantIssuedNotification = YES;
```

Comment:
I dont think you can set it this way, because there are situations where there is no email address to send the notification... and this would break that

Reply:
Have removed this and added logic in transformer